packages/build/src/plugins/node_version.ts

```diff
@@ -16,10 +16,6 @@ export type PluginsOptions = {
   inputs: Record<string, any>
 }
 
-/**
- * @deprecated will be replaced with `MINIMUM_REQUIRED_NODE_VERSION` at the 5th. of December 2022
- */
-const OLD_MINIMUM_REQUIRED_NODE_VERSION = '^12.20.0 || ^14.14.0 || >=16.0.0'
 /**
  * This node version is minimum required to run the plugins code.
  * If the users preferred Node.js version is below that we have to fall back to the system node version
```

Comment:
Let's hope that everything works, as everywhere we dropped < 4.16 but here we allow >= 4.14. I know why we did this, but just saying this is something we should keep an eye on.